### PR TITLE
add debugging trap on error to scripts with exit on err enabled

### PR DESCRIPTION
**Pull request by zmiklank**
Resolves: https://github.com/sclorg/container-common-scripts/issues/286

### CODE REVIEW

**Comment by zmiklank** [comment]
[test]

**Comment by zmiklank** [comment]
test errors are not related to this PR. But at least we can see there, that trap on ERR works ;).

File: betka.sh [hunk 1]
```diff
@@ -5,7 +5,9 @@
 # The result directory will contain generated sources from upstream directories
 #
 
-set -ex
+set -eEx
```
Comment (phracek) [inline betka.sh:8]: -x is not needed.

Comment (zmiklank) [reply betka.sh:8]: I can remove it. But the -x was present before this PR too.

Comment (zmiklank) [reply betka.sh]: It is removed in [12dc547](https://github.com/sclorg/container-common-scripts/pull/341/commits/12dc547d1bf8623b5835e16d323bd3ac36ca0724).

**Review by phracek — APPROVED** [review]
Everything LGTM. Thanks. Only nitpick. I would add there a note or comment what -E means. So the user should not Google it ;)

**Comment by zmiklank** [comment]
> Everything LGTM. Thanks. Only nitpick. I would add there a note or comment what -E means. So the user should not Google it ;)

I know, but this info can be easily gathered from `man sh`. I am not sure if it is efficient to have comments for such things here. But that's subjective opinion, of course.

**Pull request merged**